datalad/plugin/wtf.py

```diff
@@ -262,6 +293,20 @@ class WTF(Interface):
             config and metadata which could potentially contain sensitive 
             information (credentials, names, etc.).  If 'some', the fields
             which are known to be sensitive will still be masked out"""),
+        sections=Parameter(
+            args=("-S", "--section"),
+            action='append',
+            dest='sections',
+            metavar="SECTION",
+            constraints=EnsureChoice(*sorted(SECTION_CALLABLES)) | EnsureNone(),
+            doc="""section to include.  If not set, all sections. This option
+            can be given more than once to include multiple sections."""),
+        decor=Parameter(
+            args=("-D", "--decor"),
+            constraints=EnsureChoice('html_details') | EnsureNone(),
+            doc="""decoration around the rendering to facilitate embedding into
+            issues etc, e.g. use 'html_details_md' for posting collapsable entry
```

Comment:
'html_details_md' doesn't match the above value.

Reply:
thanks!